python/ray/__init__.py

```diff
@@ -24,6 +26,33 @@
     from ctypes import CDLL
     CDLL(so_path, ctypes.RTLD_GLOBAL)
 
+if (platform.system().lower() == "Linux".lower()
```

Comment:
Let's replace `"Linux".lower()` with "linux" and same for windows as it is simpler.

Reply:
I thought it might be better to keep the string literal the way it currently appears? It's less brittle if someone tries to change it later.

Reply:
sys.platform is also lowercase, not clear why uppercase would be less brittle than lowercase

Reply:
By "the way it currently appears" I was referring to the way it currently appears in the WSL implementation. Right now it's copy-pasted from there, and it's capitalized there. If it had been lowercase I'd have kept it lowercase. But anyway, it's just free-form text, and if I use lowercase and someone wants to modify it later, they can't just copy paste from the terminal anymore (and if they do, it'll be brittle, and not entirely obvious what the problem is).  Meanwhile sys.platform is more like an enum (not really textual) with well-defined symbolic values that are all lowercase, so it doesn't warrant case conversion at all. Logically they're quite different things even though physically they're strings.

Reply:
Oh wait sorry, I was thinking of the Microsoft string, not the Linux string! Let me look at this again in a second.